toolkit/javascripts/clear-filters.js

```diff
@@ -0,0 +1,36 @@
+/* 
+  ClearFilters
+
+    Requirements
+      ID: #clearAllFilters 
+      class: .dm-filters
+*/
+
+(function() {
+  "use strict";
+
+  var ClearFilters = function() {
+    this.$clearFilterLink = $('#clearAllFilters');
+    this.initaliseListener();
+  };
+
+  ClearFilters.prototype.initaliseListener = function() {
+
+    var _this = this;
+    
+    this.$clearFilterLink.on('click', function(event) {
+        event.preventDefault();
+        _this.clearFilters();
+    });
+    
+  };
+
+  ClearFilters.prototype.clearFilters = function() {
+    $('.dm-filters input[type=checkbox]:checked')
+      .prop('checked', false)
+      .trigger('change');
+  };
+
+  var clearFilters = new ClearFilters( $('#clearAllFilters') );
+  
```

Comment:
These modules are normally bound to the GOVUK or GOVUK.GDM objects. See other JS files in the frontend-toolkit for examples, but we should probably do the same with this one.